src/slm/thread/slmthread.cpp

```diff
@@ -253,8 +253,8 @@ main(int argc, char* argv[])
 
     bool usingLogPr = slm.isUseLogPr();
 
-    #define EffectivePr(a)  (usingLogPr ? ((a) / log(2.0)) : -log2f((a)))
-    #define OriginalPr(b)   (usingLogPr ? ((b) * log(2.0)) : exp2(-(b)))
+    #define EffectivePr(a)  (usingLogPr ? ((a) / log(2.0f)) : -log2f((a)))
```

Comment:
exp2f() is a C99 function , and std::exp2(float) is a C++11 feature.
C99 way is used in this PR.

If C++11 way is selected, following additional codes and C++11 compiler are also required.
``` c++
using std::log2
using std::exp2
```